doozerlib/cli/release_gen_assembly.py

```diff
@@ -29,13 +29,18 @@ def releases_gen_assembly(ctx, name):
 
 
 @releases_gen_assembly.command('from-releases', short_help='Outputs assembly metadata based on a set of specified releases')
-@click.option('--nightly', 'nightlies', metavar='NIGHTLY_NAME', default=[], multiple=True, help='A nightly release name for each architecture (e.g. 4.7.0-0.nightly-2021-07-07-214918)')
-@click.option('--standard', 'standards', metavar='4.y.z-ARCH', default=[], multiple=True, help='The name and arch of an official release (e.g. 4.8.3-x86_64) where ARCH in [x86_64, s390x, ppc64le, aarch64].')
+@click.option('--nightly', 'nightlies', metavar='NIGHTLY_NAME', default=[], multiple=True,
+              help='A nightly release name for each architecture (e.g. 4.7.0-0.nightly-2021-07-07-214918)')
+@click.option('--standard', 'standards', metavar='4.y.z-ARCH', default=[], multiple=True,
+              help='The name and arch of an official release (e.g. 4.8.3-x86_64) where ARCH in [x86_64, s390x, ppc64le, aarch64].')
 @click.option("--custom", default=False, is_flag=True,
-              help="If specified, weaker conformance criteria are applied (e.g. a nightly is not required for every arch).")
+              help="If specified, weaker conformance criteria are applied (e.g. nightly not required for every arch).")
 @click.option('--in-flight', 'in_flight', metavar='EDGE', help='An in-flight release that can upgrade to this release')
-@click.option('--previous', 'previous_list', metavar='EDGES', default=[], multiple=True, help='A list of releases that can upgrade to this release')
-@click.option('--auto-previous', 'auto_previous', is_flag=True, help='If specified, previous list is calculated from Cincinnati graph')
+@click.option('--previous', 'previous_list', metavar='EDGES', default=[], multiple=True,
+              help='A list of releases that can upgrade to this release')
+@click.option('--auto-previous', 'auto_previous', is_flag=True,
+              help='If specified, previous list is calculated from Cincinnati graph')
+@click.option('--date', is_flag=True, help='Planned release date of the assembly release. Format: YYYY-Mon-DD', required=False)
```

Comment:
Add date param

Reply:
I assume only last line was added, with the rest being refactored. I share the temptation of making a code file more readable once we're touching it, but generally speaking I'd rather segregate cosmetic changes into dedicated commits. This way, it's a lot easier for the reviewers to concentrate on what's actually changed.